app/assets/stylesheets/survey_index.scss

```diff
@@ -0,0 +1,32 @@
+@import "colors";
+@import "foundation_and_overrides";
+
+
+
+.button.success.radius.small {
+  margin-left: 170px;
+  margin-right: auto;
+  color: black;
+  background-color: darkseagreen;
```

Comment:
Color `darkseagreen` should be written in hexadecimal form as `#8fbc8f`<br>Color literals like `darkseagreen` should only be used in variable declarations; they should be referred to via variable everywhere else.